### ISSUE TITLE
Could not pass event PlayerInteractEvent to CratesPlus v4.5.3

### ISSUE THREAD

**Issue by RealOMARz** [ConnorLinfoot/CratesPlus#763]
**Please use the issue template below. Issues not using the templates will likely be closed without warning! If you are suggesting a feature you do not need to follow the template.**

If you're having config issues (such as it resetting) first make sure it's a valid YAML file using a [parser](http://yaml-online-parser.appspot.com).

### Debug Info

Due to a change in the GitHub Gist API the debug command currently doesn't work. Please can you upload your `config.yml` and provide your Bukkit version. If possible a plugin list and providing your `data.yml` is also appreciated and may be requested if needed.

Paper 1.8.8 version.

### Link/paste of stack trace/error (if any)

```
> [00:13:33 ERROR]: Could not pass event PlayerInteractEvent to CratesPlus v4.5.3
org.bukkit.event.EventException
at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:302) ~[server.jar:git-PaperSpigot-"4c7641d"]
at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[server.jar:git-PaperSpigot-"4c7641d"]
at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[server.jar:git-PaperSpigot-"4c7641d"]
at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:517) [server.jar:git-PaperSpigot-"4c7641d"]
at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:502) [server.jar:git-PaperSpigot-"4c7641d"]
at org.bukkit.craftbukkit.v1_8_R3.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:228) [server.jar:git-PaperSpigot-"4c7641d"]
at net.minecraft.server.v1_8_R3.PlayerInteractManager.interact(PlayerInteractManager.java:463) [server.jar:git-PaperSpigot-"4c7641d"]
at net.minecraft.server.v1_8_R3.PlayerConnection.a(PlayerConnection.java:763) [server.jar:git-PaperSpigot-"4c7641d"]
at net.minecraft.server.v1_8_R3.PacketPlayInBlockPlace.a(PacketPlayInBlockPlace.java:52) [server.jar:git-PaperSpigot-"4c7641d"]
at net.minecraft.server.v1_8_R3.PacketPlayInBlockPlace.a(PacketPlayInBlockPlace.java:1) [server.jar:git-PaperSpigot-"4c7641d"]
at net.minecraft.server.v1_8_R3.PlayerConnectionUtils$1.run(SourceFile:13) [server.jar:git-PaperSpigot-"4c7641d"]
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_242]
at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_242]
at net.minecraft.server.v1_8_R3.SystemUtils.a(SourceFile:44) [server.jar:git-PaperSpigot-"4c7641d"]
at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:774) [server.jar:git-PaperSpigot-"4c7641d"]
at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:378) [server.jar:git-PaperSpigot-"4c7641d"]
at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:713) [server.jar:git-PaperSpigot-"4c7641d"]
at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:616) [server.jar:git-PaperSpigot-"4c7641d"]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_242]
Caused by: java.lang.NoSuchMethodError: org.bukkit.block.Chest.getCustomName()Ljava/lang/String;
at plus.crates.Listeners.PlayerInteract.onPlayerInteract(PlayerInteract.java:42) ~[?:?]
at sun.reflect.GeneratedMethodAccessor110.invoke(Unknown Source) ~[?:?]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_242]
at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_242]
at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:300) ~[server.jar:git-PaperSpigot-"4c7641d"]
... 18 more
```

### Actions to reproduce (if known):
None. Just randomly happens in console several times.

**Comment by BramPlayzz** [comment]
Same problem.

**Comment by TheDutchEestar** [comment]
same problem. :(

**Comment by T0W0-L0L** [comment]
Same Problem :(

**Comment by LSDogX** [comment]
Saaaaaaaaaaaaaaaaaaaaaaaaaaaaaame Problem :(

**Comment by ItzCreeperBoy27** [comment]
http://www.mediafire.com/file/u8o4pnrwknf6547/Crates.jar/file heres a working cratesplus that fixes that error

**Comment by ConnorLinfoot** [comment]
Fixed in upcoming update #793
